tests/client.spec.ts

```diff
@@ -1,6 +1,10 @@
-import {client} from '../src/helpers/client';
-import {asperaDesktop} from '../src/index';
 import {mockFetch} from './mocks';
+import {asperaDesktop} from '../src';
+import {isSafari} from '../src/helpers/helpers';
+import {httpClient} from '../src/helpers/client/http-client';
+import {safariClient} from '../src/helpers/client/safari-client';
+
+const client: Client = isSafari() ? safariClient : httpClient;
```

Comment:
I'm not sure checking for Safari has any benefit in the specs. Would this ever return true?

Reply:
Done!